fala/templates/adviser/cookies.html

```diff
@@ -10,7 +10,17 @@
 
 {% set successTitle = _('Success') %}
 {% set bodyText = _('Your cookie settings were saved') %}
-{% set bodyHTML = _('<h3 class="govuk-notification-banner__heading">Your cookie settings were saved</h3><p>Other government services may set additional cookies. These will have a separate cookie policy and banner.</p><p class="govuk-body"><a class="govuk-notification-banner__link" href="/">Go back to the page you were looking at</a></p>') %}
+{% set bodyHTML %}
+  <h3 class="govuk-notification-banner__heading">
```

Comment:
@willc-work I think you will have to run `python manage.py makemessages -l cy -i locale -i venv` in order to get the previous translations applied in the `django.po` file. As your moving the order of the text around Django translations will not pick it up. 

After you've aded in translation, you then have to run python `manage.py compilemessages -l cy` to get them applied locally.

Reply:
thanks, i thought as the content was the same it would be ok but of course the line number have changed.